modos/api.py

```diff
@@ -331,23 +331,16 @@ def _add_any_element(
 
         # Copy data file to storage
         if source_file:
-            source_path = Path(source_file)
-            target_path = Path(element._get("data_path"))
-            self.storage.put(source_path, target_path)
+            # NOTE: Keep this for compatibility until ReferenceGenomes are handeled by refget
+            if isinstance(element, model.ReferenceGenome):
+                source_path = Path(source_file)
+                target_path = Path(element._get("data_path"))
+                self.storage.put(source_path, target_path)
```

Comment:
I kept this, because reference genomes would not be stored otherwise. I think with the refget PR this can be removed and will be more concise/clean. So I would just keep it for now if you agree?

Reply:
Makes sense to me, we probably want to manage them completely differently than other files in the end :+1: